tests/compiletest.rs

```diff
@@ -45,7 +46,13 @@ fn build_so_for_c_ffi_tests() -> PathBuf {
     so_file_path
 }
 
-fn run_tests(mode: Mode, path: &str, target: &str, with_dependencies: bool) -> Result<()> {
+fn run_test_config(
```

Comment:
This doesn't run the config, right? So `run` shouldn't be in the fn name IMO.

Reply:
Yea.... "run test"-config vs run "test config"